pkg/services/authz/zanzana/schema/schema_folder.fga

```diff
@@ -13,3 +13,7 @@ type folder
     define create: [user, service-account, team#member, role#assignee] or edit or create from parent
     define update: [user, service-account, team#member, role#assignee] or edit or update from parent
     define delete: [user, service-account, team#member, role#assignee] or edit or delete from parent
+    define get_permissions: [user, service-account, team#member, role#assignee] or admin or get_permissions from parent
+    define update_permissions: [user, service-account, team#member, role#assignee] or admin or get_permissions from parent
```

Comment:
```suggestion
    define update_permissions: [user, service-account, team#member, role#assignee] or admin or update_permissions from parent
```

Reply:
Oh that was not supposed to be part of this pr, thanks :+1: